### PR TITLE
Create monitored directory if does not exist

### PR DESCRIPTION
**Pull request by tecnavia-dev**
Create monitored directory, if does not exist

### CODE REVIEW

**Review by mraspaud — CHANGES_REQUESTED** [review]
Thanks for this PR. Could you write a small test for this?

File: pytroll_collectors/trollstalker.py [hunk 1]
```diff
@@ -262,11 +262,22 @@ def create_notifier(topic, instrument, posttroll_port, filepattern,
 
     # Add directories and event masks to watch manager
     for monitored_dir in monitored_dirs:
+        # Create directory, if it does not exist
```
Comment (mraspaud) [inline pytroll_collectors/trollstalker.py:265]: This comment is not needed...

**Review by mraspaud — COMMENTED** [review]
Thanks for adding this functionality, some comments inline

File: pytroll_collectors/trollstalker.py (hunk 1 above)
Comment (tecnavia-dev) [reply pytroll_collectors/trollstalker.py:265]: Ok

File: pytroll_collectors/trollstalker.py [hunk 2]
```diff
@@ -262,11 +262,22 @@ def create_notifier(topic, instrument, posttroll_port, filepattern,
...
     return notifier
 
 
+def check_if_monitored_dir_exist(directory):
+    """Create directory, if it does not exist."""
+    try:
+        os.makedirs(directory, exist_ok=True)
+    except PermissionError as err:
+        logger.error(f"Can't create monitored directory: {err}")
+        raise
```
[the remark this answers is not in the document]
Comment (tecnavia-dev) [reply pytroll_collectors/trollstalker.py]: I don't think, we need it.
I can move it, so It'll be cleaner.

**Comment by tecnavia-dev** [comment]
Do we need to make any further changes?

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**